tests/integration/config/httpbin.yaml

```diff
@@ -0,0 +1,63 @@
+# Copyright Istio Authors
```

Comment:
Can we get this file from the maistra/istio repo instead of duplicating it here?